conf/variant/common/local.conf

```diff
@@ -20,3 +20,10 @@ DIRFILES_pn-nativesdk-cmake = "1"
 
 # Don't install any extra devel packages just because we're running on QEMU
 POKYQEMUDEPS_remove = "packagegroup-core-device-devel"
+
+# Don't build .hddimg and .iso image formats as we don't need it.
+NOHDD = "1"
+NOISO = "1"
+
+# Removing tar.bz2 image format as it is not used. Whoever needs it, they can add it in respective variant's local.conf file
```

Comment:
Perhaps we can reword this as: whoever needs this will supply it in their own ``local.conf``. The variant concept is something relatively local to us.